.evergreen/config.yml

```diff
@@ -553,39 +554,21 @@ functions:
             TEST_ATLAS=1 \
             bash ${PROJECT_DIRECTORY}/.evergreen/tox.sh -m test-eg
 
-  "add aws auth variables to file":
+  "get aws auth secrets":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${aws_test_secrets_role}
     - command: shell.exec
       type: test
       params:
+        add_expansions_to_env: true
```

Comment:
These can be simplified to `include_expansions_in_env: ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY", "AWS_SESSION_TOKEN"]`.  I tried it out in https://github.com/mongodb/mongo-go-driver/pull/1365

Reply:
This particular instance still needs to be updated.  :)

Reply:
for `get_aws_secrets` it fails without `expansions_to_env`, `test-atlas` has been updated to use `include_expansions_in_env`

Reply:
Wait so there are things we're missing from the new vault?

Reply:
`get_aws_secrets` is what fetches the secrets from the vault.

Reply:
Right, okay, so now I'm confused, because that's what I'm doing in the go driver ticket.